model/src/schema/schema.ts

```diff
@@ -235,6 +235,9 @@ const feeOptionSchema = joi
     payApiKey: [joi.string().allow("").optional(), multiApiKeySchema],
     paymentReferenceFormat: [joi.string().optional()],
     payReturnUrl: joi.string().optional(),
+    allowSubmissionWithoutPayment: joi.boolean().optional().default(true),
+    maxRetries: joi.number().optional().default(3),
```

Comment:
in the runner it's looking for maxAttempts, but the joi template is looking for maxAttempts.